crates/forge/src/main.rs

```diff
@@ -79,6 +79,9 @@ struct TestArgs {
     /// Seed for the fuzzer
     #[arg(short = 's', long)]
     fuzzer_seed: Option<u64>,
+    ///Max steps
+    #[arg(short = 'm', long)]
+    max_steps: Option<u32>,
```

Comment:
I wonder if we want to have it configurable from the cli. Tbh I would even remove fuzzer config from the cli in the future, it's weird to have tests changing their internal behaviour depending on what cli args were used
cc @Arcticae

Reply:
I assume @piotmag769 that this is usefull when all your tests have a lot of steps.